source/common/stats/stats_impl.cc

```diff
@@ -149,13 +149,48 @@ bool TagExtractorImpl::extractTag(const std::string& stat_name, std::vector<Tag>
   return false;
 }
 
+RawStatData* BlockRawStatDataAllocator::alloc(const std::string& name) {
+  absl::string_view key = name;
```

Comment:
Why not just use `name` directly throughout?

Reply:
`name` comes in as a `std::string&` and we want to be able to call things like `.remove_suffix()` on it, which are `absl::string_view` features.

Reply:
ahhh, makes sense.  I didn't realize you were substringing it.